### PR TITLE
CiC CRI update env variable logic

### PR DESCRIPTION
**Pull request by Z-Mohamoud**
### What changed

Update logic to use local .env files

### Issue tracking

- [KIWI-1667](https://govukverify.atlassian.net/browse/KIWI-1667)

### Environment variables or secrets

- [ ] Documented in the README
- [ ] Added screenshots to show the implementation is working
- [ ] Ran cfn-lint on any SAM templates

### Other considerations

[KIWI-1667]: https://govukverify.atlassian.net/browse/KIWI-1667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -6,4 +6,10 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
...
 GA4_ENABLED="true"
 ANALYTICS_DOMAIN="localhost"
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Environment Variables
+IPV_STUB_URL=
+TEST_HARNESS_URL=
+API_BASE_URL=
```
Comment (Caitlin-cooling) [inline .env.sample:14]: Same q about the duplicate var

File: src/lib/config.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-require("dotenv").config();
```
Comment (Caitlin-cooling) [inline src/lib/config.js:1]: This is needed to local dev please can we add it back in

Comment (Z-Mohamoud) [reply src/lib/config.js]: Readded

File: .env.sample (hunk 1 above)
Comment (Z-Mohamoud) [reply .env.sample:14]: Updated to reflect changes made in F2F

**Review by Caitlin-cooling — APPROVED** [review]

**Pull request merged**